EIPS/eip-1202.md

```diff
@@ -0,0 +1,80 @@
+eip: 1202
+title: Voting Standard
+author: Zainan Victor Zhou <zzn@zzn.im>, and looking for co-author
+type: Standards Track
+category: ERC
+status: Draft
+created: 2018-07-08
+discussions-to: https://github.com/ethereum/EIPs/issues/1202
+---
+
+## Summary
+Propose a standard interface for voting.
+
+## Motivation
+Voting is one of the earliest example of EVM programming, and also a key to DAO/organizational governance process. We foresee many DAOs will ultimately need to leverage voting as one of the important part of their governance. By creating a voting standard for smart contract / token, we can have the following benefits
+
+### Benefits
+1. Allow general UI and applications to be built on top of a standardized voting to allow more general user to participate, and encourage more DApp and DAO to think about their governance
+2. Allow delegate voting / smart contract voting, automatic voting 
+3. Allow voting results to be recorded on-chain, in a standard way, and allow DAOs and DApps to honor the voting result programmatically.
+4. Allow the compatibility with token standard such as [ERC-20](https://eips.ethereum.org/EIPS/eip-20) or other new standards([EIP-777](https://eips.ethereum.org/EIPS/eip-777)) and item standard such as [EIP-721](https://eips.ethereum.org/EIPS/eip-721)
+5. Create massive potential for interoperability within Ethereum echo systems and other system.
+6. Allow setting voting deadline, allow determine on single or multiple options. Allow requiring voting orders. (trade-off is interface complexity, we might need [ERC-20](https://eips.ethereum.org/EIPS/eip-20) approach and later a [EIP-777](https://eips.ethereum.org/EIPS/eip-777) for advanced voting) 
+7. Recording the voting with weights with token amount.
+8. Possibly allow trust-worthy privacy-safe voting and anonymous voting (with either voter address being un-associated with the vote they cast, given a list of randomized/obfuscated voting options).
+8
+9. Possibly allow result in reward  by voting partitipation or voting result
+
+###  Use-cases: 
+1. Determine on issuing new token, issuing more token or issuing sub-token
+2. Determine on creating new item under [EIP-721](https://eips.ethereum.org/EIPS/eip-721)
+3. Determine on election on certain person or smart contract to be delegated leader for project or subproject
+4. Determine on auditing result ownership allowing migration of smart contract proxy address
+
+## Examples
+### Example 1: Simplest Version: Single Issue Yes/No Question Per Smart Contract Address Per Non-Weighted Vote
+// TODO: provide example interface and implementation
+
+### Example 2: Most Sophisticated Version: Multi-Issue Multi-Option Ordered-Result Override-able Privacy-Safe ERC-20/or other interface based Weighted Vote
+// TODO: provide example interface and implementation
+
+### Example 3: Somewhere-in-between Example 1 and Example 2 (To-Be-Discussed)
+// TODO: provide example interface and implementation
+
+## Specificiations
+// TODO
```

Comment:
Please actually have a standard before you submit a PR to add a draft to the repository.

Reply:
Thanks. I created #1237 and please take look if that is what you refer to?